stream_alert/athena_partition_refresh/main.py

```diff
@@ -338,7 +340,17 @@ def add_hive_partition(self, s3_buckets_and_keys):
                 match_dict = match.groupdict()
                 # Get the path to the objects in S3
                 path = os.path.dirname(key)
-
+                # The config does not need to store all possible tables
+                # for enabled log types because this can be inferred from
+                # the incoming S3 bucket notification.  Only enabled
+                # log types will be sending data to Firehose.
+                # This logic extracts out the name of the table from the
+                # first element in the S3 path, as that's how log types
+                # are configured to send to Firehose.
+                athena_table = path.split('/')[0] if athena_table != 'alerts' \
+                    else athena_table
```

Comment:
Since the value doesn't change in one branch, couldn't this just be
```python
if athena_table != 'alerts':
    athena_table = path.split('/')[0]
```
Easier to read IMO